examples/mass_spring_damper.md

```diff
@@ -0,0 +1,62 @@
+# Mass Spring Damper
+
+## Overview
+The mass spring damper study comprises two mass spring dampers and demonstrates how the sucessive substitution technique can be used to ensure that a co-simulation is stable. More information about successive substitution and other co-simulation stabilization techniques, please see [this paper](https://arxiv.org/pdf/1702.00686v1).
+
+
+# Example Diagram
+
+![Mass Spring Damper System](mass-spring-damper_multibody_system.png)
+
+# Example Structure
+
+There are two simulators included in the study, each representing a mass spring damper system. The first simulator calculates the mass displacement and speed of *m1* for a given force *Fk* acting on mass *m1*. The second simulator calculates force *Fk* given a displacement and speed of mass *m1*. By coupling these simulators, the evolution of the position of the two masses is computed.
```

Comment:
please use underscores.
you can use math expressions. For example, see https://into-cps-association.github.io/DTaaS/development/user/digital-twins/create.html and its markdown file